pkgs/servers/ebusd/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv, pkgs, fetchFromGitHub, argparse, mosquitto, cmake, autoconf, automake, libtool, pkg-config, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "ebusd";
```

Comment:
so sorry I just noticed this before merging actually.
this should be pname not name
it should also include a version attribute.
so most likely
```
pname = "ebusd";
version = "23.2";
```